Bottom: c123432bab33d032db1bc4d3a8ce9a11ee467eca
Top:    eff3adc820bdf274af265c3c1412bb6d547d2c0e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 20:25:43 +0100

hid/gtk: Remove duplicated prototype


---

diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index f419530..d7f0d39 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -282,7 +282,6 @@ void ghid_port_ranges_zoom (gdouble zoom);
 gboolean ghid_port_ranges_pan (gdouble x, gdouble y, gboolean relative);
 void ghid_port_ranges_scale (gboolean emit_changed);
 void ghid_port_ranges_update_ranges (void);
-void ghid_screen_update (void);
 
 gboolean ghid_note_event_location (GdkEventButton * ev);
 gboolean have_crosshair_attachments (void);
